charts/team-ns/templates/istio-gateway.yaml

```diff
@@ -24,15 +24,17 @@
 {{- end }}
 {{- $publicDomains := dict "term" $publicDomainsTerm "pass" $publicDomainsPass }}
 {{- range $tls, $tlsDomains := $publicDomains }}
-{{- if not (empty $tlsDomains) }}
+  {{- if not (empty $tlsDomains) }}
 apiVersion: networking.istio.io/v1beta1
 kind: Gateway
 metadata:
   name: team-{{ $v.teamId }}-public-tls{{ $tls }}
-  labels: {{- include "team-ns.chart-labels" $ | nindent 4 }}
+  labels:
+    apl.io/istio-gateway: default
```

Comment:
why do we need this label ?

Reply:
There is a selection on that in the operator uninstall job, so that it can work asynchronously. When there is no more canary gateway left, that means the migration is done.